tests/test_modeltest.py

```diff
@@ -94,12 +96,22 @@ def data(
     check_model(BrokenTypeModel(), should_pass=False)
 
 
+xfail_py311_pyside2 = pytest.mark.xfail(
+    sys.version_info[:2] == (3, 11) and qt_api.pytest_qt_api == "pyside2",
+    reason="Fails to OR mask flags",
+)
```

Comment:
Not sure if this is a PySide2 or Python 3.11 bug:

```pytb
>>> int(QtCore.Qt.AlignmentFlag.AlignHorizontal_Mask | QtCore.Qt.AlignmentFlag.AlignVertical_Mask)
TypeError: 'PySide2.QtCore.Qt.AlignmentFlag' object cannot be interpreted as an integer

The above exception was the direct cause of the following exception:

Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
SystemError: <class 'int'> returned a result with an exception set
```

Reply:
This `XPASS`es for me on openSUSE Python 3.11.3 and PySide2 5.15.10

Reply:
@bnavigator Thanks, #497 should help.